designer/server/src/createServer.ts

```diff
@@ -84,6 +84,16 @@ export async function createServer() {
   server.method('session.set', (id, value) => cache.set(id, value))
   server.method('session.drop', (id) => cache.drop(id))
 
+  server.method('state.get', (userId, key) => {
+    if (!userId) return Promise.reject(new Error('userId is not defined'))
+    return cache.get(`${userId}.${key}`)
+  })
+  server.method('state.set', (userId, key, value, ttl = config.sessionTtl) => {
+    if (!userId) return Promise.reject(new Error('userId is not defined'))
+    return cache.set(`${userId}.${key}`, value, ttl)
+  })
+  server.method('state.drop', (id) => cache.drop(id))
```

Comment:
I know we're not using it and it's only here for completeness, but the key needs updating?

```suggestion
  server.method('state.drop', (userId, key) => cache.drop(`${userId}.${key}`))
```